docs/the-new-architecture/landing-page.md

```diff
@@ -1,32 +1,72 @@
+Primarily, the New Architecture is an investment of the future of React Native to write polished app experiences. The New Architecture unlocks capabilities and improvements that were not possible in the legacy architecture.
 
-## Get Started with the New Architecture
+### Synchronous execution
 
-### For App Developers
+Prior, calling a native module method required passing a [callback](https://reactnative.dev/docs/next/native-modules-ios#callbacks) function to receive any data from native. Synchronous execution simplifies your native interfacing logic to provide a better experience for your app.
 
-To **create a new app** using the New Architecture, head over to [Creating a New Architecture App](use-app-template), which will get you up and running in a few quick steps with the new app template.
+In particular, synchronous rendering is a huge experience win for the New Architecture. Synchronous layout of your React views eliminates a class of bugs related to layout jumps and race conditions related to invalid layout data. Long-standing bugs with list positions and virtualization can be fixed holistically.
 
-To **migrate an existing app** to the New Architecture, follow [Adopting the New Architecture](../new-architecture-intro).
+### Support for React 18 concurrent features
 
-### For Library Maintainers
+The New Architecture brings support for [concurrent features in React](https://react.dev/blog/2022/03/29/react-v18#what-is-concurrent-react) like interruptible rendering to allow for a better user experience, especially when there are frequent UI updates.
 
-First, read up on the core concepts outlined in the [Pillars](pillars) section.
+The following compares rendering between the legacy and New Architecture in UI where the state of how many tiles to render is updated frequently by the slider value. You'll notice that the concurrent render will interrupt stale rendering work to acheive the finalized UI state faster.
 
-Then, for a **how-to guide** on supporting the New Architecture, check out the [Migration](../new-architecture-library-intro) guide.
+<table>
+<tr>
+ <td>
+<img src="https://github.com/facebook/react-native-website/assets/1309636/1cbd2d34-0639-4c20-922b-abb43e951c07" />
+</td>
+ <td>
+<img src="https://github.com/facebook/react-native-website/assets/1309636/a38cc835-0056-44e9-ab9d-4139752cd630" />
+</td>
+</tr>
 
-For information on **supporting both the Old and New Architectures**, see the [Backwards Compatibility](backward-compatibility) guide.
+</table>
+
+### Lower overhead for JS and native interoperability
+
+The New Architecture removes the use of a serializing message queue between JavaScript and native and replaces it with JavaScript Interface (JSI). JSI is an interface that allows JavaScript to hold a reference to a C++ object and vice-versa. With a memory reference, you can directly invoke methods without serialization cost.
```

Comment:
I feel like we're going out of our way to say "bridge" here but I think we should call it out because people are well aware of the bridge and we explained this as "removing the async bridge" when we [announced this project in 2018](https://reactnative.dev/blog/2018/06/14/state-of-react-native-2018).

Reply:
I agree with you here but it would technically be wrong terminology, no? 

JSI is still a _bridge_ between JS and native, just like the old JSON message queue was a _bridge_.

I definitely think people will understand it though, so maybe just put it in quotes or add a hyperlink to it that people can click to read about the bridge?

Reply:
+1 for calling out bridge here as it's part of common knowledge at this point